lib/common/index.d.ts

```diff
@@ -1,4 +1,4 @@
-export let Math: Math & {
+export class Math /*extends JSMath*/ {
```

Comment:
@zOadT do you also have any idea about this?

Reply:
if I use `class Math extend Math` it will error.

Reply:
Hmm currently not. Does the old method generate weird docs?

I can only think of manually setting the required JSMath methods for now 😕

Reply:
Oh, maybe you can just name is PlanckMath and do export {PlanckMath as Math} but I can't test it right now

Reply:
yeah, that looks great, I was also thinking about something like that

Reply:
Wait a sec, does using class actually make sense here? We won't create an instance and by inheritance we don't get the desired static methods on PlankMath, do we?

Reply:
good point, I made functions static.
I wasn't able to do `export {PlanckMath as Math}`, please feel free to try it